ruby/ql/test/library-tests/dataflow/params/params_flow.rb

```diff
@@ -135,3 +135,73 @@ def splatall(*args)
 end
 
 splatall(*[taint(69), taint(70), taint(71)])
+
+def hashSplatSideEffect(**kwargs)
+    kwargs[:p1].insert(0, kwargs[:p2])
+end
+
+kwargs = { p1: [], p2: taint(72) }
+sink(kwargs[:p1][0])
+hashSplatSideEffect(**kwargs)
+sink(kwargs[:p1][0]) # $ MISSING: hasValueFlow=72
+
+p1 = []
+sink(kwargs[:p1][0])
```

Comment:
```suggestion
sink(kwargs[:p1][0]) # $ MISSING: hasValueFlow=72
```

Reply:
Thank for spotting; the test was actually wrong. I have updated.